packages/app/src/pages/veiligheidsregio/[code]/positief-geteste-mensen.tsx

```diff
@@ -357,7 +357,13 @@ const PositivelyTestedPeople: FCWithLayout<typeof getStaticProps> = (props) => {
             source: ggdText.bronnen.rivm,
           }}
           formatTooltip={(x) => {
-            const percentage = (x[1].__value * 100) / x[0].__value;
+            const valueOne = x[0].__value;
+            const valueTwo = x[1].__value;
+
+            const percentage =
+              valueTwo === 0 && valueOne === 0
```

Comment:
```suggestion
              valueOne === 0
```

Only that check is strictly needed